### PR TITLE
Translated developer tools info page

### PR DESCRIPTION
**Pull request by springstan**
Issue affected: #3429

Description:
- added developer tools info page translations to src/translations/en.json
- updated developer-tools-info.ts and system-health-card.ts accordingly

Result:
grafik

### CODE REVIEW

File: src/panels/developer-tools/info/developer-tools-info.ts [hunk 1]
```diff
@@ -31,12 +31,18 @@ class HaPanelDevInfo extends LitElement {
...
-        : "Go to the states UI";
+        ? this.hass.localize("ui.panel.developer-tools.tabs.info.lovelace_ui")
+        : this.hass.localize("ui.panel.developer-tools.tabs.info.states_ui");
 
     const defaultPageText = `${
-      localStorage.defaultPage === OPT_IN_PANEL ? "Remove" : "Set"
-    } ${OPT_IN_PANEL} as default page on this device`;
+      localStorage.defaultPage === OPT_IN_PANEL
+        ? this.hass.localize("ui.panel.developer-tools.tabs.info.remove")
+        : this.hass.localize("ui.panel.developer-tools.tabs.info.set")
```
Comment (bramkragten) [inline src/panels/developer-tools/info/developer-tools-info.ts]: Add these as variable in the `default_ui` translation

Comment (springstan) [reply src/panels/developer-tools/info/developer-tools-info.ts]: Sure thing!

**Review by bramkragten — APPROVED** [review]
OK to merge when rebased.

**Comment by bramkragten** [comment]
That wasn't a rebase, but it does fix the merge conflict. 
For a rebase:

**If you haven't add the upstream remote yet:**
`git remote add upstream https://github.com/home-assistant/home-assistant-polymer.git`

**Rebase**
`git fetch upstream dev`  to pull the latest changes into a local dev branch
`git rebase upstream/dev` to put those changes into your feature branch before your changes and then apply your changes again

**If there are conflicts:**
Resolve the conflict(s) if there are any and save the file(s).
`git add .`
`git rebase --continue`

**Push**
After the rebase you have to force push:
`git push origin --force`

**Comment by springstan** [comment]
@bramkragten thank you very much for the explanation. I am still learning git and its terminology so your explanation is greatly appreciated :)

**Pull request merged**